### ISSUE TITLE
Issuing Let's Encrypt certificates through Rancher

### ISSUE THREAD

**Issue by HansAdema** [rancher/rancher#19538]
**What kind of request is this (question/bug/enhancement/feature request):**
Feature Request

**Steps to reproduce (least amount of steps as possible):**
N/A

**Result:**
N/A

**Other details that may be helpful:**

Rancher already offers a Certificate panel where you can upload custom SSL certificates. However, `cert-manager` offers many other ways to provide SSL certificates, notably ACME (i.e. Let's Encrypt). Sadly, Rancher does not expose these options, leaving many users having to inject `ClusterIssuer` and `Certificate` objects manually in order to make SSL issuance work.

A simple way to make this work would be to scan the existing Issuers or ClusterIssues and offering them as options in the Certificates menu. Automating the (Cluster)Issuer configuration would be a nice addition, but not required as far as I am concerned.

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
- Installation option (single install/HA): HA (Helm)

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Hosted Kubernetes from local provider
- Machine type (cloud/VM/metal) and specifications (CPU/memory): VM
- Kubernetes version (use `kubectl version`): v1.13.5
- Docker version (use `docker version`): Unknown

**Comment by AlexisDucastel** [comment]
This would be awesome. I have the need in a majority of clusters I do manage. My two cents for implementation in the Rancher UI :
- Create a certificate with empty values (and thus creating a secret without any key nor values) and use this certificate in the ingress UI with the ClusterIssuer/Issuer annotation.
- Or directly in the ingress UI, let the user specify a certificate name that does not exist that will be fulfilled by cert-manager. (or created after by the user itself).

**Comment by oliverisaac** [comment]
This is a much desired addition. It does work to create a placeholder certificate, it would be nice to not have the extra crud lying around though.

**Comment by fabn** [comment]
Also it would be nice to have a select box that lists existing Issuer for current namespace and global ClusterIssuers available in cluster

**Comment by AlexisDucastel** [comment]
At least, let the user create an empty certificate. This would allow user to select this empty certificate and let annotations do the rest of job. 

I think it would be quick and easy to do as it only requires to bypass check.

**Comment by fredleger** [comment]
I agree with @AlexisDucastel . Creating an empty certificate is certainly fast to implement and will help users to use it with some automation after.

**Comment by AlexisDucastel** [comment]
Hi, Is there any news about this feature ? 
Did we miss another way to go for having let's encrypt certificates only with Rancher UI ?
Thanks in advance
